docs/setup/native.md

```diff
@@ -6,18 +6,24 @@ Vets API requires:
 - PostgreSQL 15.x (including PostGIS 3)
 - Redis 6.2.x
 
-  The most up-to-date versions of each key dependency will be specified in the `docker-compose.yml` [file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/docker-compose.yml) and the `Dockerfile`.
+The most up-to-date versions of each key dependency will be specified in the `docker-compose.yml` [file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/docker-compose.yml) and the `Dockerfile`.
 
-  We suggest using a Ruby version manager such as [`rbenv`](https://github.com/rbenv/rbenv#installation), `asdf`, `rvm`, or `chruby` to install and maintain your version of Ruby.
+## Installing Ruby Version Manager
 
-## Installing RVM
+We suggest using a Ruby version manager such as `rbenv`, `asdf`, `rvm`, or `chruby` to install and maintain your version of Ruby.
 
-1. Install `rvm` with `brew install rvm`. This could take a while.
-2. Check the ruby version number in `.ruby-version`. Use this number to install the needed Ruby version in the command `rvm install <version_number>`. This could also take a while.
-3. Run `rvm use` within the repo to confirm that the correct version is being used.
-4. After installing a new version of Ruby, run `gem install bundler` and `bundle install` to ensure all gems are installed for the current version.
+- [rbenv](https://github.com/rbenv/rbenv)
+- [rvm](https://rvm.io/)
+- [asdf](https://asdf-vm.com/)
+- [chruby](https://github.com/postmodern/chruby)
 
-Steps 2-4 must be repeated if the repo's Ruby version is updated later.
+If the repo's Ruby version is updated later, you will need to install the newer ruby (i.e., `rvm install <version_number>`) which is located in `.ruby-version`
+
+### RVM Troubleshooting
+
+If you see an error like `Error running '__rvm_make -j10'` while installing a ruby version, this usually occurs because of a mismatch with the openssl package. 
+
+Depending on the exact error, it might be resolved by adding the openssl path for example: `rvm install 3.2.4 -C --with-openssl-dir=/opt/homebrew/opt/openssl@3.1`
```

Comment:
I believe /opt is for arm arch Macs. You might recommend the brew path prefix command as an alternative. I think this would also make it work for Linux.